jcli/src/jcli_app/block/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -66,7 +66,7 @@ initial:
   - cert: cert1qgqqqqqqqqqqqqqqqqqqq0p5avfqqmgurpe7s9k7933q0wj420jl5xqvx8lywcu5jcr7fwqa9qmdn93q4nm7c4fsay3mzeqgq3c0slnut9kns08yn2qn80famup7nvgtfuyszqzqrd4lxlt5ylplfu76p8f6ks0ggprzatp2c8rn6ev3hn9dgr38tzful4h0udlwa0536vyrrug7af9ujmrr869afs0yw9gj5x7z24l8sps3zzcmv
 
   # Initial deposits present in the blockchain
-  - legacy_fund:
-      # Legacy Cardano address
-      address: 48mDfYyQn21iyEPzCfkATEHTwZBcZJqXhRJezmswfvc6Ne89u1axXsiazmgd7SwT8VbafbVnCvyXhBSMhSkPiCezMkqHC4dmxRahRC86SknFu6JF6hwSg8
-      value: 123
+  # - legacy_fund:
+  #     # Legacy Cardano address
+  #     address: 48mDfYyQn21iyEPzCfkATEHTwZBcZJqXhRJezmswfvc6Ne89u1axXsiazmgd7SwT8VbafbVnCvyXhBSMhSkPiCezMkqHC4dmxRahRC86SknFu6JF6hwSg8
+  #     value: 123
```

Comment:
shall we just remove this since we are not handling this for now?

Reply:
I've just commented it, since it will be re-added later, but I thought about it too.